src/components/Alert.tsx

```diff
@@ -0,0 +1,10 @@
+import { FunctionComponent, ReactNode } from 'react'
+
+interface AlertProps {
+    style?: 'primary' | 'secondary' | 'success' | 'danger' | 'warning' | 'info' | 'light' | 'dark'
```

Comment:
Since `style` is a native html prop, in the md I can see the prop here being confusing. Would you mind renaming it something like `type`?

Reply:
This is a great point! I updated the prop to `type` here: d40feac.

Reply:
I'm surprised there was no linting error to warn about this. But `variant` or `type` sounds good to me!